Configuration/Eras/python/Era_Phase2C11_etlV4_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
+from Configuration.Eras.Modifier_phase2_etlV4_cff import phase2_etlV4
+
+Phase2C11_etlV4 = cms.ModifierChain(Phase2C11, phase2_etlV4)
```

Comment:
for top-level Eras, please follow the existing naming scheme:
e.g. C11 is a subdetector version for HGCal described in https://github.com/cms-sw/cmssw/tree/master/Configuration/Geometry, so use the corresponding subdetector version for MTD

Reply:
@kpedro88 as discussed above, there is no intention to add an MTD dedicated era, this is a temporary workaround while #31765 is integrated to have a variant of Phase2C11  where D72 and D73 can be exercised without problems. When that PR is integrated, there is no reason to have scenarios with PhaseC11 AND old ETL , Phase2C11 should include this modifier as a new default, and the era added here should just be retired. Do you see a problem with this? According to what I discussed with @bsunanda Phase2C11 should stay for a while, while the old ETL in conjunction with it should disappear asap

Reply:
@fabiocos thanks for clarifying. in this case, the temporary Era is acceptable as it currently is.

Reply:
@kpedro88 thanks, so I will move forward accordingly in #31765, also to make a meaningful performance test as required by @jpata . Without #31654 , there are very few hits in ETL and any performance test would be biassed